jetty-ee9/jetty-ee9-nested/src/main/java/org/eclipse/jetty/ee9/nested/ContextHandler.java

```diff
@@ -2681,6 +2681,12 @@ public class CoreContextHandler extends org.eclipse.jetty.server.handler.Context
             installBean(ContextHandler.this, true);
         }
 
+        @Override
+        protected boolean handleByContextHandler(String pathInContext, ContextRequest request, Response response, Callback callback)
+        {
+            return false;
```

Comment:
We should - meaning I should have - added a comment here as to why we don't want the nested `ContextHandler` doing the handling.